ui/src/app/services/logger.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import { Consts } from 'src/common/consts';
+import { DataService } from './data.service';
+import { map } from 'rxjs/operators';
+import { Log } from '../models/log';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoggerService extends DataService {
+
+  constructor(http: Http) {
+    super('/', http);
+  }
+
+  getLogs() {
+    let uri = Consts.API_ENDPOINT + '/logs'
+    return this.http.get(uri)
+      .pipe(
+        map(response => response.json())
+      );
+  }
+
+  logError(message) {
+    let log: Log = {
+      status: 'ERROR',
+      message: message,
+      datetime: new Date()
+    }
+    console.log(log)
+    let uri = Consts.API_ENDPOINT + '/logs'
+    return this.http.post(uri, JSON.stringify(log))
+      .pipe(
+        map(response => response.json())
+      );
+  }
+
+  logWarn(message) {
+    let log: Log = {
+      status: 'WARN',
+      message: message,
+      datetime: new Date()
+    }
+    let uri = Consts.API_ENDPOINT + '/logs'
```

Comment:
Nie dodałeś tego endpointa w api.

Reply:
@Szpedy no wiem ale to już po stronie flaska załatwiasz nie?

Reply:
Mogę załatwić zaraz xD